AIDriver.lua

```diff
@@ -416,14 +416,15 @@ end
 -- This has to be called in each update cycle to show messages
 function AIDriver:updateInfoText()
 	for msg, _ in pairs(self.activeMsgReferences) do
-		CpManager:setGlobalInfoText(self.vehicle, msg)
+		g_GlobalInfoTextHandler:setInfoText(self.vehicle, msg)
```

Comment:
Awesome, much cleaner now, can we start it with lowercase 'g'  (g_globalInfoTextHandler) to align with the Giants naming conventions?